src/file_info.rs

```diff
@@ -150,6 +182,28 @@ impl FileInfo<'_, '_, '_, '_> {
     }
 }
 
+fn _get_base_from_glob<'a>(idx: usize, glob: &'a str) -> Result<PathBuf, ConfigError> {
+    let base = if let Some(b) = glob.split('*').next() {
+        b
+    } else {
+        return Err(ConfigError::AssetGlobInvalid(
```

Comment:
I don't think this error handling is necessary; I believe that the first result of split() is always a `Some(_)`.

In this case, I would prefer to use `split_once()` as follows.

```rust
let base = match glob.split_once('*') {
    Some((former, _)) => former,
    None => glob, // (Note)
};
```
Note: This function is called after checking that it contains a wildcard character "*". So the `None` arm is not reached. But it would be reasonable to use the above code from the meaning of the function.

What do you think about it?

Reply:
Good suggestion! I didn't know about that method, so TIL. I've added tests, too, to be sure.

Thank you!